src/PathItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PathItem = ({pathName, pathItem}) => {
```

Comment:
Hmm.. I get errors when I remove the curly braces or remove the return statement